src/Symfony/Component/Asset/Package.php

```diff
@@ -23,12 +23,12 @@
  */
 class Package implements PackageInterface
 {
-    private VersionStrategyInterface $versionStrategy;
     private ContextInterface $context;
 
-    public function __construct(VersionStrategyInterface $versionStrategy, ContextInterface $context = null)
-    {
-        $this->versionStrategy = $versionStrategy;
+    public function __construct(
+        private VersionStrategyInterface $versionStrategy,
+        ContextInterface $context = null,
+    ) {
         $this->context = $context ?? new NullContext();
```

Comment:
```suggestion
        ContextInterface $context = new NullContext(),
    ) {
```

Reply:
This is a BC break, as you cannot use `null` anymore

Reply:
Ah, I see. Good thinking!